tests/foreman/ui/test_classparameters.py

```diff
@@ -0,0 +1,469 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Smart/Puppet Class Parameter"""
+
+from robottelo.decorators import run_only_on, stubbed, tier1, tier3
+from robottelo.test import UITestCase
+
+
+class SmartClassParameters(UITestCase):
```

Comment:
No-no.
Please use `SmartClassParametersTestCase` name
That is a naming convention as per https://github.com/SatelliteQE/robottelo/issues/3102